<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditGainBase</class>
 <widget class="QDialog" name="EditGainBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>284</width>
    <height>367</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MusE: Modify gain</string>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="QGroupBox" name="groupBox2">
     <property name="title">
      <string>Gain</string>
     </property>
     <layout class="QVBoxLayout">
      <item>
       <layout class="QHBoxLayout">
        <item>
         <spacer name="spacer6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>90</width>
            <height>41</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QSlider" name="sliderGain">
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>150</height>
           </size>
          </property>
          <property name="maximum">
           <number>200</number>
          </property>
          <property name="value">
           <number>100</number>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TicksBothSides</enum>
          </property>
          <property name="tickInterval">
           <number>10</number>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout">
          <item>
           <widget class="QLabel" name="textLabel2">
            <property name="text">
             <string>200%</string>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="spacer4">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Expanding</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>30</width>
              <height>109</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="textLabel1">
            <property name="text">
             <string>100%</string>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="spacer3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Expanding</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>30</width>
              <height>108</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="textLabel3">
            <property name="text">
             <string>0%</string>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="spacer7">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>51</width>
            <height>31</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout">
        <item>
         <spacer name="spacer11">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>70</width>
            <height>31</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="buttonReset">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>&amp;Reset</string>
          </property>
          <property name="shortcut">
           <string>Alt+R</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="spacer12">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>71</width>
            <height>31</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="buttonGroup2">
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout">
      <item>
       <layout class="QHBoxLayout">
        <item>
         <widget class="QPushButton" name="buttonApply">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>&amp;Apply</string>
          </property>
          <property name="shortcut">
           <string>Alt+A</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="spacer5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>61</width>
            <height>31</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="buttonCancel">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>&amp;Cancel</string>
          </property>
          <property name="shortcut">
           <string>Alt+C</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
